src/test/java/org/folio/rest/api/ReindexJobRunnerTest.java

```diff
@@ -75,6 +76,9 @@ public void canReindexInstances() {
     var lastInstanceEvent = getLastInstanceEvent(instanceId);
     assertThat(lastInstanceEvent.getPayload().getString("type"), is("REINDEX"));
     assertThat(lastInstanceEvent.getPayload().getString("tenant"), is(TENANT_ID));
+
+    assertThat(lastInstanceEvent.getHeaders(), hasEntry(
+      "reindex-job-id", job.getId()));
```

Comment:
Consider to reuse a constant rather than hardcoding `reindex-job-id`

Reply:
Done, thanks.